Project-3/Peihong_Xie_Project_3_50375781.md

```diff
@@ -0,0 +1,381 @@
+
+– Finally, computers 1, 2, and 3 are all *parts_of* network I, which is *instance_of* Object Aggregate.
+
+**[6]** Using Protege, place these in the BFO hierarchy where you think they fit best:
+```
+  (a) Bach's Well-Tempered Clavier
+  (b) Chair of the UB Philosophy Department
+  (c) SARS-CoV-2
+  (d) Mexico City
+  (e) The trunk of a minivan
+  (f) Occupation
+  (g) Ocean
+  (h) Lake
+```
+
+**Answer:** 
+
+(a) Bach's Well-Tempered Clavier is_a Generically Dependent Continunat.
+
+(b) Chair of the UB Philosophy Department is_a Role.
+
+(c) SARS-CoV-2, as a disease, is_a Disposition.
+
+(d) Mexico City is_a Site.
+
+(e) The trunk of a minivan is_a Fiat Object Part.
+
+(f) Occupation is_a Role.
+
+(g) Ocean is_a Site.
+
+(h) Lake is_a Site.
+
+**[7]** True or False; explain your answers:
+```
+  (a) An instance of Material Entity can have an instance of Immaterial Entity as part.
+  (b) An instance of Immaterial Entity can have an instance of Material Entity as part.
+  (c) An organization may have another organization as part.
+  (d) An organization may have no members as part. 
+  (e) Any site is partially bounded by some instance of Material Entity.
+  (f) A book placed under the leg of a wobbly table has acquired a new function. 
+  (g) A glass vase cushioned with packing tape for all time, has the disposition to break. 
+  (h) Spacetime is a class in BFO.
+  (i) The continuant fiat boundary class of BFO is closed, meaning, there are no subclasses beyond those identified presently in BFO. 
+```
+
+**Answer:**
+
+(a) True. Boundary or sites are in some cases **continuant parts** of their material hosts (Arp, Smith & Spear 2015: 108). For example, a digestive tube can have its *bona fide* boundary (biological membrane) as part.  
+
+(b) False. According to its definition in BFO, any immaterial entity contains no material entities as parts (Arp, Smith & Spear 2015: 107).
+
+(c) True. An organization like a company may have a fiat organization part like a labor union, which is itself an organization.
+
+(d) True. An entity can be an organization if it has members that play specific types of roles *at some time*. So the identity of an organization would not be undermined if it has no members only at some but not all times.
+
+(e) False. A site, according to its definition in BFO, may be a three-dimensional immaterial part of an immaterial entity (partially or wholly) bounded by a material entity. In this case, the site in question may be itself not partially bounded by a material entity.
+
+(f) False. A function is a disposition that exists in virtue of the bearer’s physical make-up and this physical make-up is something the bearer possesses **because it came into being**. So nothing can acquire a new function after its becoming.
```

Comment:
The explanation seems incorrect to me. The answer will be the same in either case, but being the cause of something coming into being does not define the created thing. Having a physical make-up of such-and-such as a result of being intentionally created does nothing to change the physical make-up; it only explains how that physical make-up came to occur. The physical make-up is what is relevant to the function, not what made the physical make-up come about.
To make this more concrete, it seems absurd to claim that because a book was made to contain the story of Anna Karenina that that is the only function it could ever have. As a book, it takes up physical space and has a specific weight which would be parts of it's physical make-up by necessity of it fulfilling the intended function. These parts of its physical make-up then would allow us to use it to weigh down papers, stack things on top of it, and otherwise do things with it as a physical object with these characteristics. These functions may or may not be intended, but it seems to me that they would be functions just as well as containing the story of Anna Karenina.